app/routes/article.js

```diff
@@ -0,0 +1,37 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+  session: service(),
+  store: service(),
+
+  model({ slug }) {
+    return this.store.findRecord('article', slug);
+  },
+
+  actions: {
```

Comment:
I know that we have `route-actions` in other parts of the app. I initially used them because I thought they were cool :) 
Saying that, I think that we should remove them from the app.
Could you please move these actions to the controller?
It will also clean up the code a bit. No more `this.controller.set`

Reply:
I originally wrote them in the controller and moved it to the route when I noticed the route-action. I'll move them back.

Reply:
Moved actions from route to controller, also removed `ember-route-actions-helper` since it's not used.